### DISCUSSION TITLE
【Graphql】PageQuery->list,limit field did not work as expected

### DISCUSSION THREAD

**Discussion by ijglb** [requarks/wiki#4111] [Error / Bug Report]
when i use limit field,it only return 2 pages:
![1](https://user-images.githubusercontent.com/31527814/122702959-221ff300-d283-11eb-829b-d9a2a3007f2d.png)

and when remove limit field:
![2](https://user-images.githubusercontent.com/31527814/122702974-25b37a00-d283-11eb-89fc-5b0053a3f041.png)

I have 7 page,why limit: 10 only returns 2 page? is it a bug?

Info:
Wiki.js version: 2.5.201
Node.js version: 14.17.0
PostgreSQL version: 12.7
OS: Windows Server 2012 R2

**Comment by tim-elmer** [comment]
Certainly does seem like unexpected behavior.
